spec/bulkrax/transactions/container_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Hyrax::Transactions::Container do
```

Comment:
```suggestion
# Yes, we're testing Hyrax::Transactions::Container and not Bulkrax::Transactions::Container, because we want to see the
# impact of the change on Hyrax's implementation.
RSpec.describe Hyrax::Transactions::Container do
```